public/docs/advanced/configuration.md

```diff
@@ -79,6 +79,44 @@ For each network, if unspecified, transaction options will default to the follow
 * `from`: From address used during migrations. Defaults to the first available account provided by your Ethereum client.
 * `provider`: Default web3 provider using `host` and `port` options: `new Web3.providers.HttpProvider("http://<host>:<port>")`
 
+### build_directory
```

Comment:
Probably want to change this to `contracts_build_directory` to match the property name

Reply:
Right of course. Good catch.